### PR TITLE
width/height creates 'aspect-ratio' preshint

### PR DESCRIPTION
**Pull request by tabatkins**
Testing <https://github.com/whatwg/html/pull/6032>

### CODE REVIEW

File: html/rendering/preshint-aspectratio.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        assert_equals(arImg, arControl);
+    }, "width/height attrs on <img> should form a presentational hint for 'aspect-ratio'");
+}
+
+testAr(10, 20);
+testAr(0, 1);
+testAr(1, 0);
+testAr(0, 0);
+testAr(.5, 1.5);
+</script>
```
Comment (zcorpan) [inline html/rendering/preshint-aspectratio.html:33]: Also test error handling:

* neither `width` or `height` specified
* only `width` specified
* only `height` specified
* one of them has an invalid value
* one of them has a percentage value

Comment (cbiesinger) [reply html/rendering/preshint-aspectratio.html:33]: Should also test video/canvas?

**Comment by cbiesinger** [comment]
To move this along faster, I am adding tests in https://crrev.com/c/2679289

File: html/rendering/preshint-aspectratio.html (hunk 1 above)
Comment (zcorpan) [reply html/rendering/preshint-aspectratio.html:33]: Sorry for the delay, but: yes.

I see that you added those in https://chromium-review.googlesource.com/c/chromium/src/+/2679289

**Comment by zcorpan** [comment]
https://chromium-review.googlesource.com/c/chromium/src/+/2679289 been merged and synced in #27527. Closing this. Thanks!

**Pull request closed without merging**